lib/pages/character/inventory.dart

```diff
@@ -90,42 +90,56 @@ class InventoryPage extends StatelessWidget {
     int usedSlots = inventory.length;
+    FontStyle fontStyle = (usedSlots == bag.size ? FontStyle.italic : FontStyle.normal);
     return CompanionCard(
+      padding: EdgeInsets.all(0.0),
       child: Column(
         children: <Widget>[
-          Row(
-            children: <Widget>[
-              if (bag.itemInfo != null)
-                CompanionItemBox(
-                  item: bag.itemInfo,
-                  hero: '$bagIndex ${bag.id}',
-                  size: 45.0,
-                  includeMargin: false,
-                ),
-              if (bag.itemInfo != null)
-                Expanded(
-                  child: Padding(
-                    padding: const EdgeInsets.all(8.0),
-                    child: Text(
-                      bag.itemInfo.name,
-                      style: Theme.of(context).textTheme.bodyText1.copyWith(
-                        fontWeight: FontWeight.w500
-                      )
+          Container(
+            color: Colors.indigo.shade100,
+            padding: EdgeInsets.all(8.0),
+            child: Row(
+              children: <Widget>[
+                if (bag.itemInfo != null)
+                  CompanionItemBox(
+                    item: bag.itemInfo,
+                    hero: '$bagIndex ${bag.id}',
+                    size: 45.0,
+                    includeMargin: false,
+                  ),
+                if (bag.itemInfo != null)
+                  Expanded(
+                    child: Padding(
+                      padding: const EdgeInsets.all(8.0),
+                      child: Text(
+                          bag.itemInfo.name,
+                          style: Theme
+                              .of(context)
+                              .textTheme
+                              .bodyText1
+                              .copyWith(
+                            fontWeight: FontWeight.w500,
+                            fontStyle: fontStyle
+                          )
+                      ),
                     ),
                   ),
-                ),
-              if (bag.itemInfo == null)
-                Spacer(),
-              Text(
-                '$usedSlots/${bag.size}',
-                style: Theme.of(context).textTheme.bodyText1,
-              )
-            ],
+                if (bag.itemInfo == null)
+                  Spacer(),
+                Text(
+                  '$usedSlots/${bag.size}',
+                  style: Theme.of(context).textTheme.bodyText1.copyWith(
+                    fontStyle: fontStyle
+                  ),
+                )
+              ],
+            ),
           ),
           if (usedSlots > 0)
             Container(
               width: double.infinity,
-              margin: EdgeInsets.only(top: 8.0),
+              margin: EdgeInsets.only(top: 8.0, bottom: 8.0),
+              alignment: AlignmentDirectional.center,
```

Comment:
I personally think the left alignment looks better.

Reply:
This is very much a personal thing, I've always preferred the centered style.

Reply:
Personally I'd rather have you revert this back to left alignment, so it aligns better with the bag icon.
Also change the margin to `EdgeInsets.all(8.0)`.